### DISCUSSION TITLE
Internationalization (i18n) in react js file?

### DISCUSSION THREAD

**Discussion by cvmabilin** [adonisjs/core#3775] [Help]
Hi is it possible or other work around to use the {{ t('messages.title') }} in js file for react? 

another question...

Can i get the object of a specific translation? similar to adonis v4.
In adonis v4 there is a function **Antl.get()** that can return an object of translation.

**Comment by irwing-reza** [comment]
you could use something like https://react.i18next.com/ and make use of https://www.i18next.com/how-to/add-or-load-translations#load-using-a-backend-plugin to load the JSON remotely.

You could create the endpoint on your adonis application and return the JSON.
